behaviors/services/block-storage.md

```diff
@@ -91,7 +91,7 @@ Currently a single instance per virtual chain per node.
 #### Final checks before adding
 * We assume here that the caller of this method inside the node is trusted and has already done the tests specified by `ValidateBlockForCommit`.
 * Correct block protocol version.
-* Silently discard the given block if it already exists in the database (panic if it's different from ours under this block height).
+* Silently discard the given block if it already exists in the database by comparing the actual block bytes (panic if it's different from ours under this block height).
```

Comment:
change to:

```
* Silently discard the given block if it already exists in the database by comparing the actual block bytes of the Transactions block header and Results block header only (panic if it's different from ours under this block height).
```

The approver of the block (algorithm or the node sync process) have already checked that the headers match teh body and its more redundant to recheck the entire block.